modules/viz/src/widget.cpp

```diff
@@ -113,7 +112,6 @@ void cv::viz::Widget::setRenderingProperty(int property, double value)
         case POINT_SIZE:          actor->GetProperty()->SetPointSize(float(value)); break;
         case OPACITY:             actor->GetProperty()->SetOpacity(value);          break;
         case LINE_WIDTH:          actor->GetProperty()->SetLineWidth(float(value)); break;
-        case IMMEDIATE_RENDERING: actor->GetMapper()->SetImmediateModeRendering(int(value)); break;
```

Comment:
Please use appropriate message (probably on "stderr") here that this property has no effect.

Silent removal cause old app got broken on this line:
```
CV_Assert("setPointCloudRenderingProperties: Unknown property");
```